webpack/scenes/AlternateContentSources/Create/__tests__/acsCreate.test.js

```diff
@@ -387,6 +389,7 @@ test('Can display create wizard and create simplified ACS', async (done) => {
   assertNockRequest(productScope);
   assertNockRequest(contentCredentialScope);
   assertNockRequest(smartProxyScope);
-  assertNockRequest(createScope, done);
+  assertNockRequest(createScope);
+  done();
```

Comment:
Think we have the act(done) in the tests to stop further API calls from being recorded and for nocks to get cleaned up..I am not entirely sure we need the done() here..I can try locally..

Reply:
Running `assertNockRequest(X, done);` did not work for tests, I had to change it to:
```
assertNockRequest(X);
done();
```
  In all of them